pages.tr/sunos/truss.md

```diff
@@ -0,0 +1,25 @@
+# truss
+
+> İzleme sistem çağrıları için sorun giderme aracı.
+> SunOS'in strace alternatifi.
+> Daha fazla bilgi için: <https://www.unix.com/man-page/linux/1/truss>.
+
+- Bir programı tüm alt işlemleriyle beraber çalıştırarak başlat:
+
+`truss -f {{program}}`
```

Comment:
is `program` translatable?

Reply:
Program is used as program in Turkish language.

> is `program` translatable?